plaidml/bridge/openvino/ops/deformable_convolution.cpp

```diff
@@ -0,0 +1,225 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/lib/ops.h"
+#include "plaidml/op/op.h"
+
+using namespace plaidml;           // NOLINT[build/namespaces]
+using namespace InferenceEngine;   // NOLINT[build/namespaces]
+using namespace plaidml::edsl;     // NOLINT[build/namespaces]
+using namespace plaidml::op::lib;  // NOLINT[build/namespaces]
```

Comment:
Which functions are we using from `op::lib`? I'm worried that we're going to get ODR violations if we do this. I'd suggest that whatever functions we want to use from the lib we just expose as a normal op.

Reply:
see the comment I tagged you in above